src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -97,21 +100,46 @@ async def setup_pex_cli_process(
     pex_binary: PexBinary,
     pex_env: PexEnvironment,
     python_native_code: PythonNativeCode,
+    global_options: GlobalOptions,
 ) -> Process:
     tmpdir = ".tmp"
-    downloaded_pex_bin, tmp_dir_digest = await MultiGet(
+    gets: List[Get] = [
         Get(DownloadedExternalTool, ExternalToolRequest, pex_binary.get_request(Platform.current)),
         Get(Digest, CreateDigest([Directory(f"{tmpdir}/.reserve")])),
-    )
+    ]
+    cert_args = []
+
+    # The certs file will typically not be in the repo, so we can't digest it via a PathGlobs.
+    # Instead we manually create a FileContent for it.
```

Comment:
@stuhood bump on our convo about porting `PathGlobsAndRoot` to the Async API. This is really not great to have `Path.read_bytes()` in a rule. Fine as a temporary workaround, but I think this shows the need.

We can caution plugin authors about the risks of using it. They're adults.

Reply:
Yes, we should add absolute file reads. #10360, #10769, and this usecase all require them. Opened #10842.